app/helpers/paths_helper.rb

```diff
@@ -46,6 +46,6 @@ def preview_url(_edition)
 private
 
   def should_have_auth_bypass_id?(edition)
-    %w[published archived].exclude?(edition.state) && edition.auth_bypass_id
+    %w[published archived].exclude?(edition.state) && edition.temp_auth_bypass_id
```

Comment:
It's perculiar how this project has conditionals based on `auth_bypass_id` (another one https://github.com/alphagov/publisher/pull/1313/commits/1ba31512ca20bb142c395b19f6371b88ad4c47dc#diff-8796fcc088e83cf48d2a897bfdc702e2R19) as I don't think it can ever be falsey.

Reply:
Yeah, I noticed that too, I was a bit nervous about removing them in case it changed some logic I wasn't aware of...

Reply:
Sure, I reckon we can snipe them in the follow-up PR. I left comments to that effect.